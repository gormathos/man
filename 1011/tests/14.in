sol_id;user_id;task_id;score
177019;1012;2007;81.40
177025;1018;2002;69.77
177031;1012;2001;8.11
177038;1008;2001;32.43
177039;1001;2005;75.00
177046;1002;2004;60.00
177054;1009;2001;8.11
177057;1019;2008;12.90
177060;1002;2003;59.09
177063;1002;2002;79.07
177069;1019;2003;72.73
177070;1009;2005;12.50
177073;1018;2005;81.25
177082;1014;2002;51.16
177085;1006;2002;58.14
177089;1013;2009;97.06
177096;1017;2005;12.50
177099;1006;2004;36.00
177108;1001;2006;35.90
177115;1006;2008;58.06
177122;1016;2001;24.32
177125;1004;2004;60.00
177133;1011;2007;65.12
177137;1019;2007;20.93
177147;1008;2003;0.00
177154;1017;2006;69.23
177162;1014;2004;8.00
177163;1017;2003;9.09
177166;1007;2006;74.36
177170;1001;2008;35.48
177174;1007;2005;21.88
177176;1019;2008;12.90
177182;1005;2006;35.90
177182;1001;2007;11.63
177182;1002;2002;0.00
177183;1005;2007;18.60
177193;1005;2004;0.00
177203;1004;2005;9.38
177205;1014;2001;43.24
177211;1002;2007;13.95
177221;1011;2001;2.70
177224;1006;2009;23.53
177226;1005;2006;51.28
177228;1007;2002;9.30
177237;1001;2008;100.00
177237;1017;2007;41.86
177244;1002;2006;20.51
177250;1009;2009;11.76
177253;1003;2006;5.13
177259;1017;2009;5.88
177264;1005;2008;96.77
177266;1014;2003;59.09
177269;1007;2005;6.25
177279;1014;2005;25.00
177286;1011;2008;0.00
177290;1003;2002;34.88
177292;1015;2001;97.30
177302;1004;2001;5.41
177305;1016;2003;22.73
177308;1015;2002;39.53
177311;1005;2008;100.00
177321;1014;2007;55.81
177330;1004;2005;87.50
177332;1017;2004;76.00
177338;1002;2006;15.38
177341;1012;2002;0.00
177347;1007;2007;67.44
177351;1015;2005;75.00
177357;1014;2006;2.56
177360;1012;2008;58.06
177367;1007;2004;4.00
177375;1003;2010;18.52
177381;1005;2003;95.45
177385;1005;2002;37.21
177388;1007;2001;45.95
177390;1014;2006;0.00
177390;1008;2008;19.35
177391;1014;2002;83.72
177392;1006;2004;48.00
177402;1013;2004;32.00
177407;1010;2005;56.25
177409;1008;2010;0.00
177413;1003;2003;36.36
177417;1001;2004;16.00
177423;1018;2006;100.00
177427;1008;2005;53.12
177429;1011;2001;54.05
177437;1004;2002;65.12
177440;1019;2005;62.50
177447;1008;2006;97.44
177453;1016;2009;23.53
177462;1012;2005;59.38
177470;1002;2009;79.41
177472;1005;2009;17.65
177479;1013;2008;6.45
177483;1002;2006;41.03
177492;1015;2006;28.21
177495;1017;2002;51.16
177500;1019;2001;13.51
177506;1012;2004;0.00
177506;1017;2009;38.24
177507;1011;2003;9.09
177515;1009;2004;68.00
177522;1008;2004;12.00
177530;1010;2008;70.97
177537;1019;2004;24.00
177542;1018;2004;4.00
177549;1012;2005;0.00
177552;1003;2006;76.92
177559;1019;2006;30.77
177565;1008;2009;44.12
177571;1010;2002;62.79
177579;1017;2001;72.97
177585;1003;2006;25.64
177592;1016;2008;3.23
177597;1015;2009;79.41
177605;1018;2001;89.19
177613;1012;2003;59.09
177615;1002;2008;32.26
177616;1015;2006;58.97
177626;1013;2008;41.94
177635;1015;2001;43.24
177644;1011;2008;90.32
177644;1015;2010;29.63
177649;1013;2002;60.47
177658;1016;2006;66.67
177658;1006;2009;8.82
177663;1018;2006;61.54
177668;1011;2010;25.93
177672;1001;2004;48.00
177682;1017;2005;53.12
177682;1014;2007;18.60
177684;1002;2003;18.18
177692;1011;2003;13.64
177697;1010;2003;45.45
177700;1001;2003;0.00
177704;1001;2005;50.00
177710;1010;2004;12.00
177710;1018;2008;35.48
177712;1007;2007;18.60
177715;1002;2005;3.12
177720;1007;2003;22.73
177729;1003;2009;11.76
177731;1005;2010;14.81
177736;1002;2002;48.84
177744;1011;2002;72.09
177753;1004;2001;43.24
177753;1016;2007;72.09
177760;1016;2001;70.27
177760;1008;2010;18.52
177760;1011;2009;55.88
177760;1013;2004;80.00
177767;1001;2010;59.26
177770;1001;2010;3.70
177774;1001;2001;75.68
177780;1018;2007;34.88
177787;1008;2003;18.18
177789;1018;2009;8.82
177793;1007;2008;0.00
177803;1004;2007;0.00
177812;1015;2006;51.28
177813;1010;2001;0.00
177814;1003;2001;0.00
177816;1003;2010;33.33
177822;1003;2005;0.00
177822;1014;2010;37.04
177829;1009;2001;100.00
177829;1008;2008;0.00
177839;1007;2007;95.35
177841;1009;2010;18.52
177846;1019;2005;78.12
177852;1009;2006;69.23
177857;1014;2009;20.59
177861;1007;2003;86.36
177867;1017;2004;32.00
177876;1013;2005;21.88
177882;1012;2006;25.64
177885;1007;2010;70.37
177891;1009;2009;8.82
177891;1006;2003;27.27
177896;1017;2003;27.27
177896;1018;2001;10.81
177897;1018;2003;4.55
177903;1003;2007;11.63
177904;1010;2007;86.05
177908;1011;2004;32.00
177909;1016;2001;89.19
177911;1016;2005;28.12
177911;1012;2001;40.54
177918;1010;2006;17.95
177921;1014;2002;30.23
177931;1010;2006;10.26
177938;1015;2003;31.82
177938;1010;2005;84.38
177938;1013;2003;59.09
177947;1015;2008;0.00
177953;1001;2003;50.00
177962;1013;2003;100.00
177972;1015;2007;16.28
177982;1014;2003;77.27
177983;1002;2003;45.45
177993;1002;2004;40.00
177994;1008;2008;9.68
177995;1013;2006;35.90
177999;1015;2009;38.24
178008;1015;2002;32.56
178008;1014;2005;12.50
178014;1008;2002;90.70
178014;1018;2002;13.95
178022;1013;2007;60.47
178022;1015;2007;0.00
178024;1011;2002;34.88
178028;1010;2001;32.43
178036;1010;2003;0.00
178036;1009;2002;25.58
178045;1013;2004;0.00
178053;1006;2002;39.53
178056;1013;2001;24.32
178059;1016;2003;4.55
178062;1019;2001;72.97
178063;1015;2008;38.71
178073;1004;2009;20.59
178073;1015;2007;51.16
178073;1005;2001;18.92
178082;1008;2001;2.70
178090;1010;2002;25.58
178097;1017;2001;10.81
178105;1002;2010;11.11
178115;1013;2006;82.05
178116;1004;2010;3.70
178116;1007;2001;91.89
178119;1008;2003;45.45
178123;1002;2008;16.13
178131;1007;2004;16.00
178131;1013;2003;4.55
178133;1005;2004;0.00
178135;1018;2005;56.25
178144;1017;2008;19.35
178152;1001;2001;35.14
178156;1012;2010;22.22
178158;1003;2008;19.35
178161;1017;2010;18.52
178166;1019;2009;41.18
178173;1017;2009;0.00
178181;1009;2003;18.18
178185;1012;2001;89.19
178191;1016;2007;16.28
178201;1013;2006;87.18
178206;1006;2005;6.25
178214;1018;2010;25.93
178219;1009;2004;12.00
178225;1010;2008;19.35
178235;1012;2009;79.41
178243;1006;2007;90.70
178244;1006;2006;48.72
178252;1018;2003;45.45
178257;1010;2001;5.41
178257;1008;2007;34.88
178262;1016;2003;40.91
178266;1002;2009;14.71
178272;1005;2005;15.62
178273;1013;2008;22.58
178281;1017;2007;39.53
178288;1003;2004;88.00
178298;1012;2006;46.15
178305;1001;2006;41.03
178309;1003;2001;32.43
178316;1002;2009;14.71
178319;1012;2010;18.52
178327;1003;2003;86.36
178335;1006;2010;33.33
178335;1006;2010;48.15
178339;1018;2010;51.85
178342;1012;2008;64.52
178352;1009;2003;18.18
178353;1012;2003;90.91
178354;1014;2008;0.00
178360;1013;2005;9.38
178363;1009;2008;61.29
178368;1019;2006;5.13
178371;1011;2005;59.38
178374;1013;2009;61.76
178381;1012;2007;4.65
178385;1013;2001;78.38
178388;1004;2009;91.18
178389;1010;2009;0.00
178393;1017;2006;43.59
178398;1002;2001;24.32
178408;1005;2005;18.75
178409;1014;2004;76.00
178412;1011;2006;12.82
178421;1011;2006;84.62
178424;1018;2003;27.27
178427;1018;2004;0.00
178427;1013;2010;0.00
178435;1011;2007;6.98
178435;1014;2009;100.00
178439;1017;2006;5.13
178444;1007;2010;100.00
178445;1001;2004;48.00
178454;1009;2010;88.89
178458;1007;2005;25.00
178460;1009;2009;44.12
178461;1011;2004;20.00
178471;1006;2004;24.00
178481;1005;2002;4.65
178484;1018;2006;17.95
178484;1015;2005;25.00
178490;1004;2008;54.84
178496;1008;2009;8.82
178497;1009;2004;68.00
178501;1012;2008;16.13
178503;1010;2007;25.58
178505;1007;2002;13.95
178508;1003;2009;2.94
178512;1013;2010;44.44
178520;1015;2003;9.09
178520;1010;2009;2.94
178529;1014;2004;0.00
178537;1004;2008;41.94
178542;1001;2003;27.27
178544;1014;2006;48.72
178545;1017;2008;45.16
178546;1016;2006;71.79
178554;1019;2009;55.88
178561;1013;2005;18.75
178571;1007;2008;87.10
178579;1019;2002;2.33
178580;1010;2010;88.89
178583;1006;2002;44.19
178591;1017;2008;6.45
178592;1003;2001;56.76
178598;1001;2009;11.76
178600;1016;2002;0.00
178608;1017;2003;77.27
178618;1006;2001;40.54
178622;1010;2002;69.77
178623;1003;2005;18.75
178627;1016;2004;4.00
178631;1013;2009;11.76
178638;1001;2001;24.32
178646;1004;2010;59.26
178653;1018;2007;18.60
178660;1006;2003;18.18
178666;1011;2006;15.38
178674;1017;2010;88.89
178675;1012;2010;0.00
178681;1005;2003;36.36
178691;1018;2008;87.10
178698;1008;2005;9.38
178706;1002;2005;37.50
178706;1016;2005;56.25
178706;1004;2007;25.58
178712;1003;2007;37.21
178718;1004;2004;4.00
178722;1005;2010;33.33
178730;1016;2010;11.11
178734;1018;2004;12.00
178743;1008;2009;14.71
178747;1011;2005;12.50
178756;1019;2006;69.23
178766;1003;2004;0.00
178768;1016;2006;56.41
178778;1010;2004;44.00
178786;1008;2007;62.79
178792;1005;2001;59.46
178792;1005;2007;9.30
178792;1010;2010;96.30
178799;1013;2001;16.22
178807;1001;2002;16.28
178814;1006;2006;53.85
178815;1011;2009;0.00
178822;1002;2004;0.00
178826;1012;2002;0.00
178830;1016;2010;11.11
178838;1009;2007;55.81
178846;1017;2004;28.00
178846;1008;2004;32.00
178852;1012;2009;32.35
178854;1018;2001;0.00
178855;1007;2010;0.00
178858;1016;2002;72.09
178866;1011;2003;0.00
178872;1002;2010;51.85
178876;1012;2003;0.00
178879;1003;2007;2.33
178884;1015;2002;2.33
178894;1006;2003;36.36
178896;1012;2007;30.23
178898;1019;2010;48.15
178900;1007;2006;76.92
178904;1004;2001;86.49
178904;1011;2004;0.00
178908;1015;2004;12.00
178911;1004;2008;9.68
178921;1002;2007;34.88
178922;1017;2002;34.88
178927;1003;2009;23.53
178936;1006;2007;4.65
178946;1006;2008;45.16
178950;1003;2002;74.42
178953;1001;2007;4.65
178958;1006;2005;18.75
178960;1008;2006;89.74
178966;1006;2010;40.74
178974;1017;2007;27.91
178980;1006;2006;2.56
178989;1004;2003;13.64
178996;1019;2002;11.63
178996;1016;2008;41.94
179004;1008;2010;85.19
179006;1008;2005;18.75
179010;1002;2001;5.41
179020;1009;2006;20.51
179023;1004;2002;39.53
179033;1013;2002;23.26
179043;1014;2005;100.00
179051;1004;2004;0.00
179058;1016;2004;20.00
179067;1004;2002;58.14
179068;1013;2007;95.35
179069;1015;2003;22.73
179072;1005;2009;0.00
179072;1016;2010;66.67
179075;1014;2008;32.26
179084;1004;2007;97.67
179092;1019;2010;0.00
179094;1007;2009;23.53
179100;1018;2005;100.00
179106;1014;2003;31.82
179109;1002;2005;18.75
179114;1011;2009;64.71
179118;1003;2003;86.36
179120;1019;2001;72.97
179129;1006;2001;2.70
179136;1013;2002;0.00
179139;1009;2008;3.23
179139;1016;2004;52.00
179144;1019;2008;83.87
179150;1001;2010;14.81
179158;1015;2010;77.78
179159;1006;2008;74.19
179159;1005;2003;27.27
179165;1005;2005;46.88
179167;1008;2002;4.65
179169;1003;2010;14.81
179178;1011;2008;87.10
179180;1016;2002;4.65
179185;1019;2009;73.53
179185;1019;2002;0.00
179187;1008;2006;30.77
179195;1002;2008;100.00
179200;1019;2003;13.64
179203;1006;2005;65.62
179212;1014;2007;48.84
179220;1007;2002;37.21
179230;1008;2004;0.00
179238;1012;2002;41.86
179248;1015;2010;0.00
179258;1008;2007;48.84
179258;1015;2008;74.19
179267;1009;2005;68.75
179276;1004;2003;0.00
179284;1001;2007;69.77
179292;1016;2009;2.94
179298;1013;2007;32.56
179305;1001;2008;22.58
179305;1014;2001;16.22
179309;1005;2008;51.61
179312;1004;2009;26.47
179312;1019;2007;62.79
179322;1001;2009;29.41
179331;1001;2002;18.60
179341;1004;2003;40.91
179351;1010;2003;18.18
179356;1012;2005;31.25
179357;1019;2010;44.44
179361;1009;2007;34.88
179368;1009;2001;2.70
179376;1018;2002;9.30
179376;1014;2001;62.16
179380;1018;2009;14.71
179382;1009;2002;20.93
179388;1005;2002;18.60
179393;1007;2006;35.90
179401;1003;2002;11.63
179411;1009;2007;0.00
179417;1007;2004;100.00
179425;1011;2005;0.00
179435;1010;2005;0.00
179445;1015;2004;0.00
179447;1002;2007;34.88
179456;1005;2010;11.11
179459;1015;2009;11.76
179468;1003;2008;6.45
179469;1015;2005;3.12
179469;1009;2010;3.70
179476;1013;2010;7.41
179478;1001;2006;0.00
179486;1014;2009;11.76
179487;1005;2004;52.00
179489;1003;2005;9.38
179493;1009;2003;68.18
179500;1014;2008;0.00
179505;1016;2009;94.12
179514;1004;2006;79.49
179515;1004;2010;55.56
179522;1007;2008;58.06
179532;1018;2007;34.88
179539;1010;2010;22.22
179544;1014;2010;3.70
179551;1011;2010;40.74
179560;1003;2004;36.00
179562;1007;2001;0.00
179566;1019;2007;95.35
179572;1006;2009;97.06
179576;1006;2007;18.60
179584;1010;2007;46.51
179587;1007;2009;47.06
179588;1008;2001;0.00
179594;1009;2006;30.77
179595;1014;2010;3.70
179601;1017;2001;10.81
179610;1011;2002;0.00
179616;1005;2006;0.00
179618;1018;2009;44.12
179618;1012;2009;35.29
179619;1017;2005;43.75
179626;1018;2010;7.41
179630;1012;2004;0.00
179630;1004;2006;84.62
179636;1019;2004;64.00
179644;1018;2008;80.65
179653;1005;2009;41.18
179661;1019;2004;36.00
179666;1011;2007;34.88
179676;1003;2008;19.35
179683;1016;2007;18.60
179683;1010;2008;3.23
179693;1007;2009;52.94
179698;1019;2003;0.00
179700;1004;2005;43.75
179707;1010;2004;24.00
179709;1009;2005;31.25
179714;1007;2003;4.55
179716;1001;2002;2.33
179718;1010;2009;61.76
179718;1015;2004;8.00
179721;1016;2005;43.75
179730;1011;2010;7.41
179739;1017;2010;7.41
179748;1002;2010;0.00
179755;1012;2006;28.21
179764;1012;2004;0.00
179768;1009;2008;3.23
179770;1009;2002;25.58
179770;1005;2007;16.28
179771;1001;2009;76.47
179773;1016;2008;35.48
179776;1001;2005;28.12
179778;1006;2001;70.27
179787;1017;2002;2.33
179796;1002;2001;32.43
179805;1010;2006;53.85
179811;1008;2002;46.51
179813;1004;2006;43.59
179821;1011;2001;35.14
179830;1015;2001;10.81
179831;1019;2005;40.62
179833;1005;2001;40.54
