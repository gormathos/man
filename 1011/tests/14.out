1001: Very Good [61.38%]
1002: Participant [57.53%]
1003: Participant [51.46%]
1004: Very Good [72.76%]
1005: Participant [53.54%]
1006: Very Good [64.23%]
1007: Excellent [75.28%]
1008: Participant [56.26%]
1009: Very Good [64.98%]
1010: Very Good [64.50%]
1011: Participant [57.67%]
1012: Participant [57.50%]
1013: Very Good [70.67%]
1014: Very Good [67.30%]
1015: Participant [59.72%]
1016: Very Good [65.70%]
1017: Very Good [61.39%]
1018: Very Good [63.44%]
1019: Very Good [66.96%]
